src/server/api/routers/projects.ts

```diff
@@ -305,8 +312,8 @@ export const projectsRouter = createTRPCRouter({
               ...project,
               metadata,
               metrics,
-              status,
-              nextPage: cursor + 1
+              status: "approved",
```

Comment:
This might not actually be needed anymore?

Reply:
done